README.md

```diff
@@ -1 +1,188 @@
-[Wiki](https://wiki.jenkins-ci.org/display/JENKINS/Script+Security+Plugin)
+# Script Security plugin
```

Comment:
It would be great to add some badges here:
```suggestion
# Script Security Plugin

[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/script-security)](https://plugins.jenkins.io/script-security)
[![Changelog](https://img.shields.io/github/v/tag/jenkinsci/script-security-plugin?label=changelog)](https://github.com/jenkinsci/script-security-plugin/blob/master/CHANGELOG.md)
[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/script-security?color=blue)](https://plugins.jenkins.io/script-security)
```